src/App.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import logo from './logo.svg'
+import './App.css'
+
+class App extends React.Component {
+  render() {
+    return (
+      <div className='App'>
+        <header className='App-header'>
+          <img src={logo} className='App-logo' />
+          <p>
+            Edit <code>src/App.js</code> and save to reload.
+          </p>
+          <a
+            className='App-link'
+            href='https://reactjs.org'
+            target='_blank'
```

Comment:
When creating an anchor element with `target='_blank'`, we should remember to add `rel='noopener noreferrer'` for security reasons, read more [here](https://web.dev/external-anchors-use-rel-noopener/)